google/cloud/security/inventory/pipeline_configs.yaml

```diff
@@ -0,0 +1,40 @@
+
+- resource: bigquery_datasets
```

Comment:
As a user how do I know where these resource values come from? Can we add a comment?